<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>400</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PILE OU FACE</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>141</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="connexion_layout">
     <item>
      <widget class="QLineEdit" name="connexion_pseudo"/>
     </item>
     <item>
      <widget class="QPushButton" name="connexion_button">
       <property name="text">
        <string>Connexion</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>261</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="scoreboard_layout">
     <item>
      <widget class="QLabel" name="scoreboard_title">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>SCOREBOARD</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="scoreboard_table">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>119</number>
       </attribute>
       <column>
        <property name="text">
         <string>Pseudo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Score</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>254</width>
      <height>433</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_score">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Score : 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="coeur_1">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>66</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/image/coeur.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="coeur_2">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>66</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/image/coeur.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="coeur_3">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>66</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/image/coeur.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="jeu_piece">
       <item>
        <widget class="QLabel" name="piece_texte">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piece_image">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/image/piece_face.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="button_layout_vertical">
         <item>
          <layout class="QHBoxLayout" name="button_layout_horizontal">
           <item>
            <widget class="QPushButton" name="pile_button">
             <property name="text">
              <string>Pile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="face_button">
             <property name="text">
              <string>Face</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="option">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RÃ¨gles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>60</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/restart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>connexion_pseudo</tabstop>
  <tabstop>connexion_button</tabstop>
  <tabstop>pile_button</tabstop>
  <tabstop>face_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
